src/binder/bind/bind_ddl.cpp

```diff
@@ -128,5 +147,14 @@ property_id_t Binder::bindPropertyName(TableSchema* tableSchema, const string& p
         tableSchema->tableName + " table doesn't have property: " + propertyName + ".");
 }
 
+void Binder::validatePropertyNameUniqueness(TableSchema* tableSchema, string propertyName) {
+    for (auto& property : tableSchema->properties) {
```

Comment:
Maybe I'm wrong but I remember seeing multiple use cases where you want to check if a property name exists in a tableSchema. So consider adding a tableSchema.contains(propertyName) API.

Also inline this check into bindAddProperty if it's not being reused.

Reply:
I think this is the only place where we do this check